analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.cs

```diff
@@ -49,6 +49,27 @@ void Test_2(bool condition)
             M2(o.ToString()); // Noncompliant
         }
 
+        void Test_Property()
+        {
+            MyClass o = null;
+            _ = o.MyProperty;   // Noncompliant
+        }
+
+        void Test_Field()
+        {
+            MyClass o = null;
+            _ = o.MyField;  // Noncompliant
```

Comment:
Always look on the Write side of life 🎵
I'd add a test case where a field is written:
`o.MyField = "";`

Reply:
Actually, that should be Noncompliant.